docs/ml/upload-model.md

```diff
@@ -14,12 +14,23 @@ description: "Upload a Machine Learning model to Viam to use it with the ML Mode
 The ML model service works with models trained inside and outside the Viam app.
 To use a model that you have trained yourself outside the Viam app, [upload it as a new model privately or share it in the Viam registry](#upload-a-new-model-or-new-version).
 If you need to update a previously uploaded model, you can also [upload a new version](#upload-a-new-model-or-new-version).
-To work with the `tflite_cpu` ML model service, an ML model is comprised of a <file>.tflite</file> model file which defines the model, and optionally a <file>.txt</file> labels file which provides the text labels for your model.
 
 If you have [trained](/ml/train-model/) or uploaded an ML model privately and now want to make it available for reuse, you can at any point [make the existing model public in the registry](#make-an-existing-model-public-in-the-registry).
 
 Also, [share a model from your organization](#make-an-existing-model-public-in-the-registry) with other users in the registry.
 
+## Model framework support
+
+Before uploading your model to the cloud, check and see if you will be able to deploy it to your machine.
+Model framework support is currently as follows:
+
+| Model Framework | Hardware Support | System Architecture | Description |
+| --------------- | ---------------- | ------------------- | ----------- |
+| [TensorFlow Lite](https://www.tensorflow.org/lite) | Any CPU <br> Nvidia GPU | Linux, Raspbian | Quantized version of TensorFlow that has reduced compatibility for models but supports more hardware. |
```

Comment:
not sure if System Architecture column is accurate: asked question [here](https://docs.google.com/document/d/1IBL2H2IL618YZuR_FSAe6FzA5_hbmqQ-fmrd_xBHLOg/edit?disco=AAABJEf7MMI)

Reply:
@npentrel I also made a ticket here https://viam.atlassian.net/browse/DOCS-1980 -- table is not fully complete as support is not complete

Reply:
OK - maybe we comment out the ones that aren't supported yet and then uncomment once supported?

Reply:
I think as they're all supported (just support to be extended further for PyTorch and TensorFlow) I will keep but update?